.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,14 @@
+blank_issues_enabled: true
+contact_links:
+    - name: Become a SPONSOR ❤
+      url: https://github.com/sponsors/trvswgnr
+      about: Help this project by becoming a sponsor
+    - name: Read our CODE OF CONDUCT doc
+      url: https://github.com/trvswgnr/sloth-pipe?tab=readme-ov-file
+      about: This link will take you to our CODE OF CONDUCT doc
+    - name: Read our CONTRIBUTION GUIDE
+      url: https://github.com/trvswgnr/sloth-pipe?tab=readme-ov-file#contributing
+      about: This link will take you to our CONTRIBUTION GUIDE doc
+    - name: New? Know how to contribute to open source
+      url: https://opensource.guide/
+      about: Get some more ideas on open source
```

Comment:
We might add a contribution guide as well for later, I wouldn't add one just yet.